.env.test

```
@@ -4,3 +4,6 @@ PG_PORT=5432
 PG_USERNAME=postgres
 PG_PASSWORD=1234
 PG_HOST=localhost
+
+## JWT Secret
+JWT_SECRET=EnterSecretKey
```

Minor comment here but we can generate a real JWT secret here that we just won't use anywhere else, just so that its realistic for the test env.

Ran the command `openssl genrsa 2048` to get the private key below 👍 
```suggestion
JWT_SECRET="-----BEGIN RSA PRIVATE KEY-----
MIIEowIBAAKCAQEAyjHm+yAPav/Uu96oD5cCsvlZwLkvLzRXyYyRxGay7zouIkTm
oBr4DghXWdBdR8URkBbGfC8mIri8IiJ/bMtNvY87DBGv4m6KUL6Eay0JeVL7seu/...
-----END RSA PRIVATE KEY-----"
```